diff --git a/node_modules/@tldraw/editor/dist-cjs/lib/license/LicenseProvider.js b/node_modules/@tldraw/editor/dist-cjs/lib/license/LicenseProvider.js
index f411fec..2fa70dc 100644
--- a/node_modules/@tldraw/editor/dist-cjs/lib/license/LicenseProvider.js
+++ b/node_modules/@tldraw/editor/dist-cjs/lib/license/LicenseProvider.js
@@ -31,8 +31,8 @@ var import_LicenseManager = require("./LicenseManager");
 const import_meta = {};
 const LicenseContext = (0, import_react.createContext)({});
 const useLicenseContext = () => (0, import_react.useContext)(LicenseContext);
-function shouldHideEditorAfterDelay(licenseState) {
-  return licenseState === "expired" || licenseState === "unlicensed-production";
+function shouldHideEditorAfterDelay(_licenseState) {
+  return false;
 }
 const LICENSE_TIMEOUT = 5e3;
 function LicenseProvider({
diff --git a/node_modules/@tldraw/editor/dist-esm/lib/license/LicenseProvider.mjs b/node_modules/@tldraw/editor/dist-esm/lib/license/LicenseProvider.mjs
index 6b50326..5ed6f09 100644
--- a/node_modules/@tldraw/editor/dist-esm/lib/license/LicenseProvider.mjs
+++ b/node_modules/@tldraw/editor/dist-esm/lib/license/LicenseProvider.mjs
@@ -4,8 +4,8 @@ import { createContext, useContext, useEffect, useState } from "react";
 import { LicenseManager } from "./LicenseManager.mjs";
 const LicenseContext = createContext({});
 const useLicenseContext = () => useContext(LicenseContext);
-function shouldHideEditorAfterDelay(licenseState) {
-  return licenseState === "expired" || licenseState === "unlicensed-production";
+function shouldHideEditorAfterDelay(_licenseState) {
+  return false;
 }
 const LICENSE_TIMEOUT = 5e3;
 function LicenseProvider({
